charts/liquibase-migration/values.yaml

```diff
@@ -1,53 +1,106 @@
+  migrationImage:
+  # -- (string) Image tag to use for the migration image; **warning** that setting this can cause inconsistent migrations (default recommended)
   # @default -- Set to global.applicationVersion
-  imageTag: null
-  # migration.serviceAccount -- (string) App's k8s service account
-  # WARNING: If not applied earlier than migration.syncWave, selective sync may be necessary in cold-start scenarios
-  # @default -- Set to "${appName}-sa"
-  serviceAccount: null
-  # migration.appContainerShell -- (string) Shell executable to use in the app's container
-  appContainerShell: "bash"
-  # migration.liquibaseCommand -- (string) Liquibase CLI command, can be "updateSQL" for a no-op dry-run 
-  liquibaseCommand: "update"
-  # migration.ignoreLiquibaseFailures -- (bool) If the Liquibase CLI's exit code should be ignored
-  ignoreLiquibaseFailures: false
+  migrationImageTag:
+  # -- Docker command directive to invoke a shell in the container, to expand migrationArgs* values
+  migrationShell: ["bash", "-c"]
+  # -- Time to wait before attempting to start Liquibase, to allow proxy to boot
+  migrationDelay: "15s"
+  # -- (list) Java classpath location(s) containing Liquibase, JDBC driver, any packaged changelogs, and all dependencies;
+  # if the app is packaged then its `.jar` is sufficient (CLI argument wrapped in double quotes, bash expansion allowed)
+  migrationArgsClasspath:
+  # -- If the Kubernetes job should always succeed regardless of the Liquibase command's exit status
+  migrationArgsLiquibaseCommand: "update"
+  # -- (string) Java classpath location of the Liquibase changelog file, expanded by migrationShell
+  migrationArgsConfigChangelog: null
+  # -- (string) Java class name of the JDBC driver to use, expanded by migrationShell
+  migrationArgsConfigDriver: null
+  # -- (string) JDBC URL of the database, expanded by migrationShell
+  migrationArgsConfigUrl: null
+  # -- (string) Optional arbitrary additional arguments to be passed, from https://docs.liquibase.com/commands/home.html
+  migrationArgsAdditional: null
+  # -- Controls for mounting and passing a Liquibase.properties file (necessary unless all other migrationConfigArgs* values passed)
+  # @default -- With the secret name omitted, no configuration file will be mounted or passed
+  migrationConfigFileMount:
+    # -- (string) The exact name of a Kubernetes secret containing the desired file under a file-named key
+    secretName: null
+    # -- The full mount path of the file, ending with the file itself
+    mountFilePath: "/etc/liquibase.properties"
+  # -- Controls for setting up database authentication, one sub-object must be provided;
+  # existingKubernetesSecret takes precedence if provided
+  # @default -- None
+  migrationDatabaseCredentials:
+    # -- Use an existing Kubernetes secret for credentials directly
+    # @default -- With the secret name omitted, no existing Kubernetes secret will be accessed
+    existingKubernetesSecret:
```

Comment:
nit: should this be `fromKubernetesSecret` for consistency with `fromVaultSecret`?